### PR TITLE
📖 amp-base-carousel: Add `controls` attribute to component documentation

### PR DESCRIPTION
**Pull request by caroqliu**
Partial for #28284.

### CODE REVIEW

File: extensions/amp-base-carousel/amp-base-carousel.md [hunk 1]
```diff
@@ -128,6 +128,14 @@ useful when using `visible-count`. This
 
 #### Miscellaneous
 
+##### controls
+
+Either `"always"`, `"auto"`, or `"never"`, defaults to `"auto"`. This determines if and when prev/next navigational arrows are displayed. Note: When `outset-arrows` is `true`, the arrows are shown `"always"`.
+
+-   `always`: Arrows are always displayed.
+-   `auto`: Arrows are displayed when a mouse has interacted with the carousel, and
```
Comment (micajuine-ho) [inline extensions/amp-base-carousel/amp-base-carousel.md:136]: nit: carousel.

**Review by micajuine-ho — APPROVED** [review]
Curious, do you know why `controls` was never documented?

File: extensions/amp-base-carousel/amp-base-carousel.md (hunk 1 above)
Comment (caroqliu) [reply extensions/amp-base-carousel/amp-base-carousel.md:136]: Good catch! Updating this in the other md file I copied this from as well. 😂

Comment (caroqliu) [reply extensions/amp-base-carousel/amp-base-carousel.md:136]: Actually there was more to this sentence so added the rest now.

**Comment by caroqliu** [comment]
> Curious, do you know why `controls` was never documented?

No clue unfortunately. But it is used openly in manual samples and valid AMP, so I don't see why it should not be documented. I assume it was an oversight.

**Pull request merged**